jena-rdfconnection/src/main/java/org/apache/jena/rdflink/dataset/assembler/VocabAssemblerHTTP.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.jena.rdflink.dataset.assembler;
+
+import org.apache.jena.assembler.Assembler;
+import org.apache.jena.assembler.assemblers.AssemblerGroup;
+import org.apache.jena.rdf.model.Property;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.sparql.core.assembler.AssemblerUtils;
+import org.apache.jena.system.Vocab;
+
+public class VocabAssemblerHTTP
+{
+    private static final String NS = "http://jena.apache.org/2025/http#";
```

Comment:
Does this need to be a new namespace? Are there name clashes?

(it is the usually problem of large namespaces vs remembering name namspaces)